### PR TITLE
Pilots models with options + camera changes

### PR DESCRIPTION
**Pull request by mixMugz**
- Added GLTF file with two nodes PILOT_0, PILOT_1.
- Added interior model definition to XML with VISIBILITY code for pilot's nodes.
- Added FMC OPTIONS MISC page with switching ability of pilots.
- Added macro replacements in main module (from A32NX source).
- Changes in camera definition.
  1. Repositioning Pilot/Copilot camera
  2. Modified Close View/Landing View camera
  3. Modified All quickView camera
  4. Changed Instruments camera
    - Ctrl-1 (correct visible PFD)
    - Ctrl-2 (correct MCP camera. Recenter)
    - Ctrl-3 (correct position)
    - Ctrl-4 (tune positioning, moved up to avoid speed braker lever)
    - Ctrl-5/6 (no changes)
    - Ctrl-7 (show full pedestal)
    - Ctrl-8 (show full overhead)
    - Ctrl-9 (show full cockpit behind pilots)
    - Ctrl-0 (show EFB for future purpose)

### CODE REVIEW

**Comment by ninjomcs** [comment]
where did you get the models for the pilots? is this your own model?

**Comment by mixMugz** [comment]
> where did you get the models for the pilots? is this your own model?

It's default Asobo models (ingame) what is used for external view. GLTF contain only empty NODES. Game place pilots automaticly.

**Comment by DesertRick13** [comment]
Got the mod on Flightim.to. Really appreciate the co-pilot. Haven't played with the rest yet. I think it would be cool for direct inclusion.

File: html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_MainDisplay.js [hunk 1]
```diff
@@ -1197,6 +1210,24 @@ class B747_8_FMC_MainDisplay extends Boeing_FMC {
...
+                }
+            });
+        });
+        this._labelElements.forEach((row) => {
+            row.forEach((column) => {
+                if (column !== null) {
+                    column.innerHTML = this._formatCell(column.innerHTML);
+                }
+            });
+        });
```
Comment (ninjomcs) [inline html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_MainDisplay.js:1230]: would you be able to specify exactly what changes this introduces?

Comment (mixMugz) [reply html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/FMC/B747_8_FMC_MainDisplay.js:1230]: > would you be able to specify exactly what changes this introduces?

It's allow replace short defined macro to correct property:

Defined macro:
`    _formatCell(str) {
        return str
            .replace(/{white}/g, "<span class='white'>")
            .replace(/{blue}/g, "<span class='blue'>")
            .replace(/{yellow}/g, "<span class='yellow'>")
            .replace(/{green}/g, "<span class='green'>")
            .replace(/{red}/g, "<span class='red'>")
            .replace(/{inop}/g, "<span class='inop'>")
            .replace(/{small}/g, "<span class='s-text'>")
            .replace(/{sp}/g, "&nbsp;")
            .replace(/{end}/g, "</span>");
    }` 
    
    Call for coloring:
    `displayCurrentPilotsOption = `{small}NONE{end}/{green}PILOT{end}/{small}COPILOT{end}/{small}BOTH{end}``

**Pull request merged**